src/components/card/card.component.jsx

```diff
@@ -0,0 +1,32 @@
+import "./card.style.css";
+function Card({ title, data, size }) {
```

Comment:
Also we can pass the size as size.toLowerCase() in case of anyone pass it `Big` of `Small` acceditly it would be a frustrated bug that one may not notice easily.